var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -193,7 +193,10 @@ def nvariant_bool(feature):
             '-DVTK_USE_SYSTEM_HDF5:BOOL=%s' % variant_bool('+hdf5'),
             '-DVTK_USE_SYSTEM_JPEG:BOOL=ON',
             '-DVTK_USE_SYSTEM_LIBXML2:BOOL=ON',
-            '-DVTK_USE_SYSTEM_MPI4PY:BOOL=%s' % variant_bool('+python+mpi'),
```

Comment:
This can be simplified if you move it down into the "enable python" block, then you only need to check `if '+mpi' in spec`

Reply:
Also while you're at it, probably should add an else clause in the python block below that explicitly disabled python.